monai/transforms/utility/dictionary.py

```diff
@@ -409,6 +413,7 @@ def __call__(
         return d
 
 
+@deprecated(since="0.6.0", removed="0.7.0", msg_suffix="Please consider using `EnsureTensord` transform instead.")
```

Comment:
Why do we need to deprecate this? If so, I think that implies that we should also be deprecating ToNumpy, ToPIL, ToCupy etc

Reply:
Yes, you are making a good point.
Let me remove the deprecated tag.

Thanks.